ui/lib/pki/package.json

```diff
@@ -8,6 +8,7 @@
     "ember-cli-babel": "*",
     "ember-cli-htmlbars": "*",
     "ember-cli-typescript": "*",
+    "@hashicorp/design-system-components": "2.3.1",
```

Comment:
per Jordan's conversation in the dev channel - do we want this to have a `*` so the main app manages the version?